nodebuilder/p2p/flags.go

```diff
@@ -24,6 +30,11 @@ Such connection is immune to peer scoring slashing and connection manager trimmi
 Peers must bidirectionally point to each other. (Format: multiformats.io/multiaddr)
 `,
 	)
+	flags.String(
+		nodeNetworkFlag,
+		"",
+		"The name of the network to connect to, e.g. "+listProvidedNetworks(),
```

Comment:
We should specify here that `p2p.network` should only be passed on `start` and will not have a persisted effect if only passed on `init`

Reply:
We don't need to specify that as this is default behavior for all the flags

Reply:
Wait a minute - passed flags on `init` have a persisted effect, but not on `start` @Wondertan - that's the default behaviour, not the vice versa

Reply:
I was worried about flags' conceptual dependences on init/start semantics, but I def misread your comment. Adding the note here makes sense to me now

Reply:
done